statesman-engine/src/main/java/io/appform/statesman/engine/handlebars/HandleBarsHelperRegistry.java

```diff
@@ -1165,6 +1192,23 @@ private void registerValueFromJsonString() {
         });
     }
 
+    private void registerSHA256() {
+        handlebars.registerHelper("sha256",(Integer context, Options options) -> {
```

Comment:
Why not take Object here and use return of context.toString() to generate sha256?